lib/api/api_auth.go

```diff
@@ -42,6 +42,74 @@ type authAndSessionMiddleware struct {
 	evLogger   events.Logger
 }
 
+func basicAuthAndSessionMiddleware(cookieName string, guiCfg config.GUIConfiguration, ldapCfg config.LDAPConfiguration, next http.Handler, evLogger events.Logger) http.Handler {
```

Comment:
This code was added back by mistake, it should not be in use, please remove again.